Umbraco-Uno/Widgets/Text/index.md

```diff
@@ -4,22 +4,29 @@ versionFrom: 8.0.0
 
 # Text widget
 
-![Text widget icon](images/The-Text-Widget1.png)
-
-The Text Widget lets you create a text field on the page. The widget contains a Rich text editor and the option for adding sidebar widgets, which can be text areas, links and/or buttons. These are explained in more detail in the [Sidebar widgets](../Sidebar-widgets) article.
+The Text widget lets you create a text field on the page. The widget contains a Rich text editor and the option for adding sidebar widgets, which can be text areas, links and/or buttons. These are explained in more detail in the [Sidebar widgets](../Sidebar-widgets) article.
 
 This widget is useful if you only need to add a text box. Most of the other widgets come with an option to add text as well.
 
 ## Sample
 
-![Text Frontend](images/Text-Frontend1.png)
+![Text Frontend](images/Text-front.png)
 
 ## Configuration options
 
+You will have the options to choose the size of your text box and to add a sidebar widget to it.
+
+![Text Backoffice](images/Text-final.png)
+
+### Content
+
 - Content (Rich Text Editor)
 - Sidebar widgets
 
-![Text Backoffice](images/Text-Backoffice1.png)
+### Settings
+
+- Container Width
+- Sidebar Left
 
 ::: Note
 The Text widget is the *standard* width per default. This cannot be changed.
```

Comment:
You say earlier in this article that "you will have the option to choose the size of your text box".
Doesn't that contradict what it says in this box? 🤔